api/actions/v1alpha1/filters_types.go

```diff
@@ -0,0 +1,27 @@
+package v1alpha1
+
+import "github.com/odigos-io/odigos/api/k8sconsts"
+
+const ActionNameFilters = "Filters"
+
+// FiltersConfig defines the configuration for the Filters action.
+type FiltersConfig struct {
+	// Attributes is a map of attribute keys and values to filter on.
+	Attributes map[string]string `json:"attributes,omitempty"`
```

Comment:
consider making it explicit what attributes means (it can be any of span attributes, span event attributes, metric attributes, data point attributes, log record attributes).